logfire/_internal/integrations/httpx.py

```diff
@@ -148,26 +148,30 @@ def instrument_httpx(
             )
 
         tracer_provider = final_kwargs['tracer_provider']
-        instrumentor.instrument_client(client, tracer_provider, request_hook, response_hook)  # type: ignore[reportArgumentType]
+        instrumentor.instrument_client(client, tracer_provider, request_hook, response_hook)
 
 
 class LogfireHttpxInfoMixin:
-    headers: httpx.Headers
+    headers: httpx.Headers | None
```

Comment:
Rather make this an empty Headers object than let it be None. Definitely don't raise errors.